enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -955,6 +1000,22 @@ class AdjointGenerator
     Builder2.setFastMathFlags(getFast());
   }
 
+  inline void getForwardBuilder(IRBuilder<> &Builder2) {
```

Comment:
If you're going to make this new helper function, I'd suggest also reworking all the existing code that creates a forward builder (usually named BuilderZ or something) to use the helper as well.

Reply:
I could only find two such instances of BuilderZ, moved the helpers to GradientUtils and made those instances use getForwardBuilder